README.md

```diff
@@ -19,3 +19,19 @@ steps:
       appSecret: ${{ secrets.APP_SECRET }}
       apkPath: 'path/to/apk'
 ```
+
+### All Params
+| Name | Description | Default | Optional
```

Comment:
I think it makes it clearer to rephrase this as `Required`. You'll need to update the table accordingly. 

```suggestion
| Name | Description | Default | Required
```